arbitrum-docs/launch-orbit-chain/orbit-sdk/introduction.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Arbitrum Orbit SDK'
+sidebar_label: 'Introduction'
+description: 'Arbitrum SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 0
+---
+
+# Welcome to the Arbitrum Orbit SDK
+
+Dive into the forefront of decentralized blockchain development with the Arbitrum Orbit SDK, a comprehensive toolkit designed for developers eager to explore the innovative Arbitrum Orbit platform. This SDK facilitates the creation and management of your own Orbit chain using the advanced capabilities of Viem, a modern alternative to traditional Ethereum libraries like ethers.js and web3.js.
+
+## Arbitrum Orbit chains
+
+With the Arbitrum Orbit SDK, you have the flexibility to establish your Orbit chain as a Layer 3 (L3) chain settling transactions with any Layer 2 (L2) chains, or as a Layer 2 (L2) chain directly settling on Layer 1 (L1) chains. This adaptability opens up a realm of possibilities for application development and blockchain use cases.
+
+### Enhanced Development Experience
+
+The SDK is developed using TypeScript, providing a strong foundation for your development efforts. At the core of our SDK is Viem, which elevates the development experience by offering a streamlined, efficient, and modular approach to interacting with Ethereum's blockchain and smart contracts. Whether you're a seasoned developer familiar with ethers.js or new to blockchain development, Viem and the Arbitrum Orbit SDK offer a smooth transition and easy comparison to understand the benefits and similarities between the platforms, especially with resources like [Viem's ethers.js migration guide](https://viem.sh/docs/ethers-migration.html).
+
+### Capabilities of the SDK:
+- **Configuration and Deployment**: Simplify the process of configuring and deploying the core contracts needed for an Orbit chain.
+- **Initialization and Management**: After deployment, utilize the SDK to initialize your chain and adjust configuration parameters as necessary.
+- **Ongoing Support and Updates**: Look forward to future releases that will bring new features, such as tools for creating custom dashboards and monitoring systems for your Orbit chain.
+
+By choosing the Arbitrum Orbit SDK, you're not just accessing a set of development tools; you're unlocking a universe of opportunities to create bespoke blockchain projects tailored to your needs. Stay tuned for more exciting updates and features as we continue to enhance the Arbitrum Orbit SDK, making it an even more powerful resource for developers.
+
+Below, you will find detailed information about the steps for setting up an Orbit chain and how to utilize the Orbit SDK throughout the process:
```

Comment:
```suggestion
## Deployment Process
	
	You can utilize the Orbit SDK to set up an Orbit chain with the following steps:
```

Reply:
The comment is outdated but taken into account by trimming transition sentences in the latest edit.